css/style.css

```diff
@@ -57,6 +58,60 @@ a {
 }
 
 
+/* open and close nav menu */
+
+.nav-menu .closeMenu i {
+  font-size: 2rem;
+  display: none;
+  cursor: pointer;
+}
+
+.navbar-list .mainMenu {
+  height: 100vh;
+  position: fixed;
+  top: 0;
+  right: 0;
+  left: 0;
+  z-index: 10;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  background: fixed;
+  background-color: #000;
+  transition: top 1s ease;
+  display: none;
+}
```

Comment:
- Please, kindly ensure your mobile menu and contents match the Figma design as much as you can below:

*your design*
![n](https://user-images.githubusercontent.com/59397159/179748243-b17137b8-c4fc-4fc1-8cad-567f6d829329.png)
* figma design*
![match](https://user-images.githubusercontent.com/59397159/179748247-d82bcea9-ef7d-41a1-9c46-3b73d311da41.png)

Reply:
*README FILE*

> Please, kindly update your readme by doing the following to make your readme more professional.
- Add a "Getting started" header and add some instructions on how users can have your project up and running locally on their machine
- Add the "Acknowledgement" header and ad its content giving appropriate credit to Figma, for using their design template.
- Update the MIT.md file and add the licensed content as of now, the link is broken.
- Update the description of your portfolio to be more descriptive as what you have now doesn't say anything about your portfolio.